X/Xyce/build_tarballs.jl

```diff
@@ -37,11 +36,11 @@ make install
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-   
+
 platforms = filter(p -> (!Sys.iswindows(p) &&
                          !Sys.isapple(p) &&
-                         !Sys.isfreebsd(p))
-                   , supported_platforms())
+                         !Sys.isfreebsd(p)),
+                   supported_platforms())
```

Comment:
Any particular reason for not using
```julia
platforms = filter(Sys.islinux, supported_platforms())
```
?

Reply:
To be very clear which platforms are not supported. But yeah, when it's one entire OS and nothing else this can be used.

Reply:
Is there a way I make this

> 
> platforms = filter(Sys.islinux, supported_platforms())

without running the entire pipeline again? (That is an intense run)

Reply:
Not really, because we cache the builds using as index the git tree sha of the project, so if we edit the file the cache will inevitably be invalidated.  If there are no other substantial changes planned, we can merge this

Reply:
None from my side